backend/audit/templates/audit/submission-progress.html

```diff
@@ -114,7 +114,7 @@ <h1 class="border-base-light margin-top-0">{{ auditee_name | default_if_none:''
 					<a class="usa-link text-bold"
 						href="/report_submission/federal-awards/{{ report_id }}">Upload the Federal Awards workbook</a>
 					<p class="margin-0">
-						Provide general information on the auditee and auditor, financial statements for the auditee entity, and information on the entity's federal program compliance.
+						This workbook should be completed by the auditee. For each federal award received, you'll need the financial and agency details. This is also were you list the number of audit findings.
```

Comment:
`were` --> `where`?

Reply:
Good catch, thanks @jperson1! I've fixed that typo.